subsys/bluetooth/host/smp.h

```diff
@@ -142,6 +142,9 @@ int bt_smp_br_send_pairing_req(struct bt_conn *conn);
 int bt_smp_init(void);
 
 int bt_smp_auth_cb_overlay(struct bt_conn *conn, const struct bt_conn_auth_cb *cb);
+#if defined(CONFIG_BT_PASSKEY_KEYPRESS_API)
+int bt_smp_auth_keypress_notify(struct bt_conn *conn, enum bt_conn_auth_keypress type);
+#endif
```

Comment:
I don't see any need to guard this either

Reply:
If `enum bt_conn_auth_keypress` is guarded, then this must be as well.